crates/plex-api/src/server/mod.rs

```diff
@@ -44,19 +46,94 @@ impl Server {
         }
     }
 
+    // pub async fn metadata(&self, key: &str) -> Result<MetadataMediaContainer> {
+    //     let mut response = self
+    //         .client
+    //         .get(key)
+    //         .header("Accept", "application/json")
+    //         .send()
+    //         .await?;
+
+    //     match response.status() {
+    //         StatusCode::OK => {
+    //             let text = response.text().await?;
+
+    //             match from_str::<MediaContainerWrapper<MetadataMediaContainer>>(&text) {
+    //                 Ok(w) => Ok(w.media_container),
+    //                 Err(e) => {
+    //                     eprintln!("{}", text);
+    //                     Err(e.into())
+    //                 }
+    //             }
+    //         }
+    //         _ => Err(crate::Error::from_response(response).await),
+    //     }
+    // }
```

Comment:
What are your plans for this block?

Reply:
Oh I was just using that during testing, I forgot it was there, I'll remove it.